src/storage/config.rs

```diff
@@ -381,7 +381,7 @@ mod tests {
         let mut cfg = Config::default();
         assert!(cfg.validate().is_ok());
 
-        let max_pool_size = std::cmp::max(1, SysQuota::cpu_cores_quota() as usize);
+        let max_pool_size = std::cmp::max(4, SysQuota::cpu_cores_quota() as usize);
```

Comment:
Why the change?

Reply:
the original test logic is wrong, change the logic to make the test logic right. 
See: https://github.com/tikv/tikv/blob/1fb8980ccab9ff40c1adc206df52952dab8e8ad8/src/storage/config.rs#L119-L128